debian/compat

```diff
@@ -0,0 +1 @@
+9
```

Comment:
Would it be easy to make it compatible with Debian8/Jessie too? Not for this PR, but later if easy to do...

Reply:
It could be done, but debian 8 only has packages for tango 8. Possible if tango 9 was backported, or the dependencies were removed.